### ISSUE TITLE
打印的日志为什么只有save model in而没有其他信息呢

### ISSUE THREAD

**Issue by Vincy9501** [PaddlePaddle/PaddleOCR#9800]
[2023/04/23 12:45:36] ppocr INFO: save model in ./output/rec/r34_vd_none_bilstm_ctc/latest
[2023/04/23 12:45:36] ppocr INFO: save model in ./output/rec/r34_vd_none_bilstm_ctc/latest
[2023/04/23 12:45:36] ppocr INFO: save model in ./output/rec/r34_vd_none_bilstm_ctc/iter_epoch_801
[2023/04/23 12:45:37] ppocr INFO: save model in ./output/rec/r34_vd_none_bilstm_ctc/latest
epoch、global_step什么的都没有，最后acc=0。
使用的自己的数据集，预训练模型rec_r34_vd_none_bilstm_ctc，配置文件rec_r34_vd_none_bilstm_ctc.yml
Global:
  use_gpu: true
  epoch_num: 1200
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/rec/r34_vd_none_bilstm_ctc/
  save_epoch_step: 3
  # evaluation is run every 2000 iterations
  eval_batch_step: [0, 2000]
  cal_metric_during_train: True
  pretrained_model: ./pretrain_models/rec_r34_vd_none_bilstm_ctc/best_accuracy
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/imgs_words_en/word_10.png
  # for data or label process
  character_dict_path: ppocr/utils/feifang_dict.txt
  character_type: en
  max_text_length: 25
  infer_mode: False
  use_space_char: False
  save_res_path: ./output/rec/predicts_r34_vd_none_bilstm_ctc.txt

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.0001
  regularizer:
    name: 'L2'
    factor: 0

Architecture:
  model_type: rec
  algorithm: CRNN
  Transform:
  Backbone:
    name: ResNet
    layers: 34
  Neck:
    name: SequenceEncoder
    encoder_type: rnn
    hidden_size: 256
  Head:
    name: CTCHead
    fc_decay: 0

Loss:
  name: CTCLoss

PostProcess:
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: SimpleDataSet
    data_dir: E:/OCR/PaddleOCR/train_data/rec/
    label_file_list: ["E:/OCR/PaddleOCR/train_data/rec/train.txt"] 
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 100]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    batch_size_per_card: 256
    drop_last: True
    num_workers: 8

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: E:/OCR/PaddleOCR/train_data/rec/
    label_file_list: ["E:/OCR/PaddleOCR/train_data/rec/val.txt"] 
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 100]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 256
    num_workers: 4

**Comment by MissPenguin** [comment]
是不是数据量太少了，用默认的配置和数据集能正常跑吗
